tests/beaker_tests/cisco_fabricpath_global/test_fabricpath_global.rb

```diff
@@ -238,14 +238,16 @@ def test_harness_fabricpath_global(tests, id)
   tests[id][:ensure] = nil
 end
 
+def testbed_cleanup(agent)
+  remove_all_vlans(agent)
```

Comment:
Does this cleanup work on n5k and n6k since fabric path is supported on those platforms?